<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="description"
        content="AMix-1 Homepage: A Pathway to Test-Time Scalable Protein Foundation Model"> <!-- TODO: add some description, visible outside -->
  <meta name="keywords" content="AMix-1, Protein Foundation Model"> <!-- TODO: add some keywords for search engine -->
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>AMix-1 Homepage: A Pathway to Test-Time Scalable Protein Foundation Model</title>



  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-PYVRSFMDRL"></script>
  <script>
    window.dataLayer = window.dataLayer || [];

    function gtag() {
      dataLayer.push(arguments);
    }

    gtag('js', new Date());

    gtag('config', 'G-PYVRSFMDRL');
  </script>
  <script async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>

  <link href="https://fonts.googleapis.com/css?family=Google+Sans|Noto+Sans|Castoro"
        rel="stylesheet">

  <link rel="stylesheet" href="./static/css/bulma.min.css">
  <link rel="stylesheet" href="./static/css/bulma-carousel.min.css">
  <link rel="stylesheet" href="./static/css/bulma-slider.min.css">
  <link rel="stylesheet" href="./static/css/fontawesome_6_7_2.all.css">
  <link rel="stylesheet"
        href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <link rel="stylesheet" href="./static/css/index.css">

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script defer src="./static/js/fontawesome_6_7_2.all.js"></script>
  <script src="./static/js/bulma-carousel.min.js"></script>
  <script src="./static/js/bulma-slider.min.js"></script>
  <script src="./static/js/index.js"></script>
</head>
<body>

<nav class="navbar" role="navigation" aria-label="main navigation">
  <div class="navbar-brand">
    <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
      <span aria-hidden="true"></span>
      <span aria-hidden="true"></span>
      <span aria-hidden="true"></span>
    </a>
  </div>
  <!-- <div class="navbar-menu">
    <div class="navbar-start" style="flex-grow: 1; justify-content: center;">
      <a class="navbar-item" href="">
      <span class="icon">
          <i class="fas fa-home"></i>
      </span>
      </a> -->

      <!-- <div class="navbar-item has-dropdown is-hoverable"> -->
        <!-- <a class="navbar-link">
          More Research
        </a> -->
        <!-- <div class="navbar-dropdown"> -->
          <!-- <a class="navbar-item" href="https://hypernerf.github.io">
            HyperNeRF
          </a> -->
        <!-- </div> -->
      <!-- </div> -->
    <!-- </div> -->

  <!-- </div> -->
</nav>


<section class="hero">
  <div class="hero-body">
    <div class="container is-max-desktop">
      <div class="columns is-centered">
        <div class="column has-text-centered">
          <h1 class="title is-1 publication-title">
            AMix-1: A Pathway to Test-Time Scalable<br>Protein Foundation Model
          </h1> <!-- TODO: fix the title -->
          <!-- <div class="is-size-5 publication-authors">
            <span class="author-block">
              <a href="">Author 1</a><sup>1,2,3</sup>,</span>
            <span class="author-block">
              <a href="">Author 2</a><sup>1,2,3</sup>,</span>
            <span class="author-block">
              <a href="">Author 3</a><sup>1,2,3</sup>,
            </span>
          </div> -->

          <div class="is-size-5 publication-authors">
            <span class="author-block"><sup>1</sup>Shanghai Artificial Intelligence Laboratory</span>
            <!-- &nbsp; -->
             <br>
            <span class="author-block"><sup>2</sup>Generative Symbolic Intelligence Lab (GenSI), Tsinghua University</span>
            <br>
            <span class="author-block"><sup>3</sup>Institute for AI Industry Research (AIR), Tsinghua University</span>
            <!-- <br> -->
            <!-- <span class="author-block"><sup>4</sup>SIA-Lab of Tsinghua AIR and ByteDance Seed</span> -->
          </div>
          <div class="is-size-6 publication-authors">
            <span class="author-block">*Full author and affiliation list in Contributions</span>
          </div>
            <div class="publication-links">
              <!-- PDF Link. -->
              <span class="link-block">
                <a href="https://arxiv.org/pdf/2507.08920"
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="fas fa-file-pdf"  style="color: #ff5c5c;"></i>
                  </span>
                  <span>Paper</span>
                </a>
              </span>
              <span class="link-block">
                <a href="https://arxiv.org/abs/2507.08920"
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="ai ai-arxiv"  style="color: #ff5c5c;"></i>
                  </span>
                  <span>arXiv</span>
                </a>
              </span>
              <!-- Arxiv Link. -->
              <!-- <span class="link-block">
                <a href=""
                   class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                      <i class="ai ai-arxiv"></i>
                  </span>
                  <span>arXiv</span>
                </a>
              </span> -->
              <!-- Code Link. -->
              <!-- <span class="link-block"> -->
                <!-- <a href="https://github.com/BytedTsinghua-SIA/DAPO" -->
                   <!-- class="external-link button is-normal is-rounded is-dark"> TODO: fix repo link -->
                  <!-- <span class="icon"> -->
                      <!-- <i class="fab fa-github"></i> -->
                  <!-- </span> -->
                  <!-- <span>Code</span> -->
                  <!-- </a> -->
              <!-- </span> -->
              <!-- Dataset Link. -->
              <!-- <span class="link-block"> -->
                <!-- <a href="https://huggingface.co/datasets/BytedTsinghua-SIA/DAPO-Math-17k" -->
                   <!-- class="external-link button is-normal is-rounded is-dark"> TODO: fix dataset link -->
                  <!-- <span class="icon"> -->
                      <!-- <i class="fa-regular fa-database"></i> -->
                  <!-- </span> -->
                  <!-- <span>Data</span> -->
                  <!-- </a> -->
              <!-- </span> -->
              <!-- Model Link. -->
              <span class="link-block">
                <a href="https://huggingface.co/GenSI"
                   class="external-link button is-normal is-rounded is-dark"> <!-- TODO: fix model link -->
                  <span class="icon">
                    <!-- <i class="fa-solid fa-face-smiling-hands"></i> -->
                    <i class="fa-solid fa-face-smiling-hands" style="color: #FFD43B;"></i>
                  </span>
                  <span>Model (Incoming)</span>
                  </a>
              </span>
              <span class="link-block">
                  <a href="#Citation" class="external-link button is-normal is-rounded is-dark">
                    <span class="icon">
                      ðŸ“š
                    </span>
                    <span>Citation</span>
                  </a>
                </span>
            </div>
        </div>
      </div>
          
      <div class="columns is-centered has-text-centered">
        <div class="column is-full">
          <h2 class="subtitle has-text-centered">
            <img src="./static/images/main.png" style="max-width: 100%; height: auto; width: 100%;"/>
          </h2>
        </div>
      </div>
          <p>
            We introduce <b>AMix-1</b>, a powerful <b>protein foundation model</b> built on Bayesian Flow Networks and empowered by a systematic training methodology, encompassing pretraining scaling laws, emergent capability analysis, in-context learning strategy, and test-time scaling algorithm. 
            To guarantee robust scalability, we establish a predictive, <b>noise-aware scaling law</b> and reveals the progressive <b>emergence of structural understanding</b> via loss perspective, culminating in a strong <b>1.7-billion model</b>. 
            Building on this foundation, we devise a multiple sequence alignment (MSA) based <b>in-context learning</b> strategy to unify protein design into a general framework, where AMix-1 recognizes deep evolutionary signals among MSAs and consistently generates structurally and functionally coherent proteins. 
            This framework enables the successful design of a dramatically improved AmeR variant with an up to 50x activity increase over its wild type. 
            Pushing the boundaries of protein engineering, we further empower AMix-1 with an <b>evolutionary test-time scaling</b> algorithm for \textit{in silico} direction evolution that delivers substantial, scalable performance gains as verification budgets are intensified, laying the groundwork for next-generation lab-in-the-loop protein design.
          </p>
    </div>
  </div>
</section>

<section class="section">
  <div class="container is-max-desktop">
    <!-- Abstract -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-full">
        <h2 class="title is-3">Scaling Law</h2>
        <div class="content has-text-justified">
          <p>
            We are the first to characterize scaling law and emergent ability for an advanced generative model, i.e. Bayesian Flow Network, and pretrain a family of powerful models, AMix-1, scaling from 8 million to 1.7 billion parameters.
            Following figures illustrate scaling laws of cross-entropy and training FLOPs across various model sizes and the noise level \(\alpha\). We show the equation between \(\mathcal{L}\) and FLOPs \(F\), indicating that the AMix-1 models have predictive scaling laws.
          </p>
        </div>
        <div class="columns is-centered has-text-centered">
        <div class="column is-full">
          <h2 class="subtitle has-text-centered">
            <img src="./static/images/ScalingLaw.png" style="max-width: 100%; height: auto; width: 100%;"/>
          </h2>
        </div>
      </div>
    </div>

  </div>
</section>


<section class="section">
  <div class="container is-max-desktop">
    <!-- Abstract -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-full">
        <h2 class="title is-3">Emergent Ability</h2>
        <div class="content has-text-justified">
          <p>
            Emergent abilities of AMix-1 models under varying noise levels \(\alpha\). Noise levels of \(\alpha=0.16\) and \(0.32\) exhibit clear emergence. Darker colors indicate models with more parameters. Lower cross-entropy, and consequently stronger downstream performance, emerges as the model size increases
            .
          </p>
        </div>
        <div class="columns is-centered has-text-centered">
        <div class="column is-full">
          <h2 class="subtitle has-text-centered">
            <img src="./static/images/Emergent.png" style="max-width: 100%; height: auto; width: 100%;"/>
          </h2>
        </div>
      </div>
    </div>

  </div>
</section>

<section class="section">
  <div class="container is-max-desktop">
    <!-- Abstract -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-full">
        <h2 class="title is-3">In-Context Learning</h2>
        <div class="content has-text-justified">
          <p>
            We establish a unified protein design framework leveraging AMix-1's <b>in-context learning</b> ability and evolutionary signals in multiple sequence alignments for <b> novel structure- and function-preserving generation</b>.
            Following figure illustrate the comparison of standard LLM in-context learning (top) with protein family conditioning via MSA profiles in AMix-1 (bottom). In LLM ICL, few-shot exemplars are token-level demonstrations; in AMix-1, the MSA profile serves as a <b>dense, position-wise family prompt</b>.
          </p>
        </div>
        <div class="columns is-centered has-text-centered">
        <div class="column is-full">
          <h2 class="subtitle has-text-centered">
            <img src="./static/images/ICL.png" style="max-width: 100%; height: auto; width: 100%;"/>
          </h2>
        <div class="columns is-vcentered">
            <div class="column is-half">
              <p class="has-text-justified">
                We design a high-activity AmeR variant with a \(50\times\) improvement over the wild type, showcasing AMix-1's real-world impact on functional protein engineering.
                Specifically, we used AMix-1 to generate 40 candidate protein sequences, each containing no more than ten mutations. The functional activity of these variants was measured using a fluorescent reporter assay, where higher DNA-binding affinity leads to stronger repression of a fluorescent protein, quantified as "fold repression." A higher fold repression value indicates better function. AMix-1's performance was compared to three baselines: Wild-Type (WT), a single-mutant library, and EvoAI, a state-of-the-art directed evolution method. The illustration on the right shows the wet-lab verification process guided iteratively by AMix-1.
              </p>
            </div>
            <div class="column is-half">
              <img src="./static/images/wet_lab.png" style="max-width: 100%; height: auto; width: 100%;"/>
            </div>
          </div>
      </div>
    </div>

  </div>
</section>

<section class="section">
  <div class="container is-max-desktop">
    <!-- Abstract -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-full">
        <h2 class="title is-3">Test-Time Scaling</h2>
        <div class="content has-text-justified">
          <p>
            With the in-context learning ability of AMix-1, we push this capacity even further: introducing a novel <b>evolutionary test-time scaling</b> algorithm that uses AMix-1 as the proposer and an external plug-and-play verifier to generate candidate sequences from evolutionary constraints, achieving directed protein evolution through arbitrary in-silico verifiers or experimental screening.
          </p>
        </div>
        <div class="columns is-centered has-text-centered">
        <div class="column is-full">
          <h2 class="subtitle has-text-centered">
            <img src="./static/images/TTS.png" style="max-width: 100%; height: auto; width: 100%;"/>
          </h2>
          <h2 class="subtitle has-text-centered">
            <img src="./static/images/TTS_results.png" style="max-width: 100%; height: auto; width: 100%;"/>
          </h2>
        </div>
      </div>
    </div>

  </div>
</section>

<section class="section" id="Citation">
  <div class="container is-max-desktop">
    <!-- Abstract -->
    <div class="columns is-centered has-text-centered">
      <div class="column is-full">
        <h2 class="title is-3">Citation</h2>
        <div class="content has-text-justified">
          <p>
            If you find this work useful, please cite our paper:</p>
        </div>
        <div class="columns is-centered has-text-centered">
        <div class="column is-full">
          <pre style="text-align: left;"><code>
@article{lv2025amix1,
  title={AMix-1: A Pathway to Test-Time Scalable Protein Foundation Model},
  author={Changze Lv*, Jiang Zhou*, Siyu Long*, Lihao Wang, Jiangtao Feng, Dongyu Xue, Yu Pei, Hao Wang, Zherui Zhang, Yuchen Cai, Zhiqiang Gao, Ziyuan Ma, Jiakai Hu, Chaochen Gao, Jingjing Gong, Yuxuan Song, Shuyi Zhang, Xiaoqing Zheng, Deyi Xiong, Lei Bai, Ya-Qin Zhang, Wei-Ying Ma, Bowen Zhou, Hao Zhou},
  journal={arXiv preprint arXiv:2507.08920},
  year={2025}
}</code>  </pre>
        </div>
      </div>
      <div class="columns is-centered has-text-centered">
        <div class="column is-full" style="display: none;">
          <script type="text/javascript" id="clustrmaps" src="//clustrmaps.com/map_v2.js?d=bR5O65Pq9kfxkXsRX5bpXbHJlS_ts6MErHP2YIkPXDk&cl=ffffff&w=a"></script>
        </div>
    </div>

  </div>
</section>

<!-- <section class="section">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Contributions</h2>
        <div class="content has-text-justified">
          <p>
            
          </p>
        </div>
      </div>
    </div>
  </div>
</section> -->

<!-- <section class="section">
  <div class="container is-max-desktop">
    <div class="columns is-centered">
      <div class="column is-full-width">
        <h2 class="title is-3">Related Links</h2>
        <div class="content has-text-justified">
          
        </div>
      </div>
    </div>
  </div>
</section> -->


<!-- <section class="section" id="BibTeX">
  <div class="container is-max-desktop content">
    <h2 class="title">BibTeX</h2>
    <pre><code>@article{2025puffinzero,
                author    = {Author 1, Author 2, Author 3},
                title     = {Puffin-Zero},
                journal   = {Arxiv},
                year      = {2025},
              }</code></pre>
  </div>
</section> -->


<footer class="footer">
  <div class="container">
    <div class="content has-text-centered">
      <!-- <a class="icon-link"
         href="./static/pdf/AMix-1_paper.pdf">
        <i class="fa-solid fa-file-pdf" style="color: #ec4646;"></i>
      </a> -->
      <!-- <a class="icon-link" href="" class="external-link" disabled>
        <i class="fab fa-github"></i>
      </a> -->
    </div>
    <div class="columns is-centered">
      <div class="column is-8">
        <div class="content">
          <p>
            This website is adapted from <a href="https://nerfies.github.io/">Nerfies</a> and licensed under a <a rel="license"
                                                href="http://creativecommons.org/licenses/by-sa/4.0/">Creative
            Commons Attribution-ShareAlike 4.0 International License</a>.
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>
</body>
</html>
